external-crates/move/crates/move-compiler/src/parser/ast.rs

```diff
@@ -165,18 +165,69 @@ pub struct UseDecl {
 //**************************************************************************************************
 
 #[derive(Debug, Clone, PartialEq, Eq)]
+#[allow(clippy::large_enum_variant)]
 pub enum AttributeValue_ {
     Value(Value),
     ModuleAccess(NameAccessChain),
 }
+
 pub type AttributeValue = Spanned<AttributeValue_>;
 
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub enum Attribute_ {
+pub enum ParsedAttribute_ {
     Name(Name),
     Assigned(Name, Box<AttributeValue>),
-    Parameterized(Name, Attributes),
+    Parameterized(Name, Spanned<Vec<ParsedAttribute>>),
+}
+
+pub type ParsedAttribute = Spanned<ParsedAttribute_>;
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum ExpectedFailureKind_ {
+    Empty,
+    Name(Name),
+    MajorStatus(Value),
+    AbortCode(AttributeValue),
+}
+
+pub type ExpectedFailureKind = Spanned<ExpectedFailureKind_>;
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+#[allow(clippy::large_enum_variant)]
+pub enum Attribute_ {
+    BytecodeInstruction,
+    DefinesPrimitive(Name),
+    Deprecation {
+        note: Option<Symbol>,
+    },
+    Error {
+        code: Option<Value>,
+    },
+    External {
+        attrs: Spanned<Vec<ParsedAttribute>>,
+    },
+    Syntax {
+        kind: Name,
+    },
+    VerifyOnly,
```

Comment:
Do you think we can just kill this? Or probably not since if anyone was using it we will now generate code for it? I am sad

Reply:
There will be another PR on top of this one adding modes, and this one will go away.